### PR TITLE
Adding Script component for inline script :)

### PR DESCRIPTION
**Pull request by sanjaiyan-dev**
Using `<Script />` component from `next/script` for inline colour mode script.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -95,7 +95,5 @@
     "recursive-readdir": "^2.2.1",
     "unist-util-map": "^1.0.3"
   },
-  "resolutions": {
-    "**/sharp": "^0.30.3"
-  }
+  "packageManager": "yarn@3.2.1"
```
Comment (eps1lon) [inline package.json:98]: This is creating a large diff in the lockfile. This project requires yarn v1. If you use that for install you should not have any changes to the lockfile.

**Comment by eps1lon** [comment]
> Using <Script /> component from next/script for inline colour mode script.

Can you elaborate what behavior you're seeing without this change, what behavior the change results in, and why we should change it?

**Comment by eps1lon** [comment]
> Sorry if I made any mistakes raised_handsblush

I didn't mean to imply that. I was wondering what this change is doing in the first place or what it's intended to do.

File: package.json (hunk 1 above)
Comment (sanjaiyan-dev) [reply package.json:98]: > This is creating a large diff in the lockfile. This project requires yarn v1. If you use that for install you should not have any changes to the lockfile.

Sorry, I think this error might happened while running `next:lint` :(

**Comment by sanjaiyan-dev** [comment]
> > Sorry if I made any mistakes raised_handsblush
> 
> I didn't mean to imply that. I was wondering what this change is doing in the first place or what it's intended to do.

And i thought it is a best practise to use `next/script` :)

**Pull request closed without merging**